plugins/inventory/kubevirt.py

```diff
@@ -317,6 +320,28 @@ def get_port_from_service(service: Dict) -> Optional[str]:
 
         return None
 
+    @staticmethod
+    def is_windows(guest_os_info: Dict, annotations: Dict) -> bool:
+        if (id := guest_os_info.get("id")) is not None:
+            return id == "mswindows"
+
+        if (
+            cluster_preference := annotations.get(
+                ANNOTATION_KUBEVIRT_IO_CLUSTER_PREFERENCE_NAME
+            )
+        ) is not None:
+            return cluster_preference.startswith("windows")
```

Comment:
create the var `windows = "windows"` to avoid repeating the same string?

Reply:
I'd like to stick to the way it is. IMO it fits the rest of the code style in the file.

Reply:
All right!